theme/static/js/hs_resource_table.js

```diff
@@ -914,4 +939,22 @@ $.fn.dataTable.ext.search.push (
 
         return inLabels && inFilters && inGrantors;
     }
-);
\ No newline at end of file
+);
+
+function are_restrictions_passed() {
+    let input_label_value = $('#txtLabelName').val().trim();
+    return !(input_label_value.length === 0 ||
+        has_long_single_word(input_label_value) ||
+        input_label_value.length > 64 ||
+        input_label_value.indexOf('<') > -1 ||
+        input_label_value.indexOf('>') > -1
+    )
+}
+
+function has_long_single_word(input_str) {
+    return input_str.indexOf(' ') < 0 && input_str.length > 32;
+}
+
+function get_illegal_tag_name_message() {
+    return "Tag names are invalid: it can't be empty, contain special characters < or >, or have a single word longer than 32 characters.";
```

Comment:
We could also escape the HTML so we do not have to restrict the usage of certain characters. Here is a solution I like for escaping HTML characters in javascript:
```
var entityMap = {
    '&': '&amp;',
    '<': '&lt;',
    '>': '&gt;',
    '"': '&quot;',
    "'": '&#39;',
    '/': '&#x2F;',
    '`': '&#x60;',
    '=': '&#x3D;'
  };

  function escapeHtml (string) {
    return String(string).replace(/[&<>"'`=\/]/g, function fromEntityMap (s) {
      return entityMap[s];
    });
  }
```
From https://github.com/janl/mustache.js/blob/master/mustache.js#L73

Reply:
With the two changes above, and adding logic to disable the Create button when the input is empty, we wouldn't need a warning message at all and the interface would be more intuitive.

Reply:
I like the css fix. 

Regarding naughty characters, @Lizabrazil and I did some research. We found no user tickets that complain about using html tags, nor long single word. This implies our users don't use the tag the way the github issue stated.

Reply:
It is indeed an edge case, but I do think it is better to take care of it in the right way rather than preventing it by adding rules to what a user can input.

Reply:
I will double check with @Lizabrazil tomorrow if we should continue spending effort on the issue since our users don't hit the scenario.

Reply:
@Maurier thank you spending time providing a good feedback solution.